### ISSUE TITLE
Bug located in Stackup3D

### ISSUE THREAD

**Issue by Devin-Crawford** [ansys/pyaedt#2977]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Conductors are of type dielectric. This makes all metal objects "solve_inside".

### Steps To Reproduce

Run the machine learning patch example for HFSS.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

All dependencies installed.

**Comment by dcrawforAtAnsys** [comment]
Line 350 of stackup_3d.py
```
self._duplicated_material = self.duplicate_parametrize_material(material_name)
```
makes the cloned material dielectric even if it is copper.

**Comment by dcrawforAtAnsys** [comment]
Error is in ``stackup_3d.DuplicateParameterizedMaterial``

**Comment by dcrawforAtAnsys** [comment]
Digging deeper, the error is in ``Material.is_conductor()``.  When conductivity is a parameter (i.e. a string) the Boolean 
```python
float(cond)
```
cannot be evaluated. Somehow the expression for the conductivity value needs to be evaluated.

**Comment by dcrawforAtAnsys** [comment]
@pyAnsysUser the ML example was removed as there were multiple errors. The problem with "solve inside" has been found and addressed. I'll create a PR soon with a simple example of a patch to demonstrate usage.

**Comment by dcrawforAtAnsys** [comment]
Should be resolved in PR #3566.
